cmake/public/LoadHIP.cmake

```diff
@@ -62,11 +62,8 @@ ENDIF()
 # THRUST_PATH
 IF(DEFINED ENV{THRUST_PATH})
   SET(THRUST_PATH $ENV{THRUST_PATH})
-ELSEIF(DEFINED ENV{THRUST_ROOT})
-  # TODO: Remove support of THRUST_ROOT environment variable
-  SET(THRUST_PATH $ENV{THRUST_ROOT})
 ELSE()
-  SET(THRUST_PATH ${ROCM_PATH}/Thrust)
+  SET(THRUST_PATH ${ROCM_PATH}/include)
```

Comment:
Are you sure this is correct? ("include" seems to be the header include path, not the root path)

Reply:
Yes, that's correct. The hip-thrust package is putting headers into `/opt/rocm/include/thrust/` so in order to access them with `#include <thrust/something.h`, we need that.